engine/common/oncall_gateway/tasks.py

```diff
@@ -98,3 +99,102 @@ def delete_slack_connector_async_v2(**kwargs):
     except Exception as e:
         task_logger.error(f"Failed to delete SlackConnectorV2 oncall_org_id={oncall_org_id} exc={e}")
         raise e
+
+
+# New tasks to use once chatops v3 is landed
+@shared_dedicated_queue_retry_task(
+    autoretry_for=(Exception,),
+    retry_backoff=True,
+    max_retries=100,
```

Comment:
nit: isn't 100 too much considering it uses exponential backoff? maybe have like 10?

Reply:
I choose 100 retries initially to be sure, that we will get the alert about retrying task in the monitoring, since it might affect user's chatops experience. Probably it's better to have some counter metric, but I don't want to change that in  this PR to limit the amount of changes.